packages/google_maps_flutter/google_maps_flutter/test/fake_maps_controllers.dart

```diff
@@ -126,29 +127,21 @@ class FakePlatformGoogleMap {
     markersToChange = _deserializeMarkers(markerUpdates['markersToChange']);
   }
 
-  Set<MarkerId> _deserializeMarkerIds(List<dynamic> markerIds) {
+  Set<MarkerId> _deserializeMarkerIds(List<dynamic>? markerIds) {
     if (markerIds == null) {
-      // TODO(iskakaushik): Remove this when collection literals makes it to stable.
-      // https://github.com/flutter/flutter/issues/28312
-      // ignore: prefer_collection_literals
-      return Set<MarkerId>();
+      return <MarkerId>{};
     }
     return markerIds.map((dynamic markerId) => MarkerId(markerId)).toSet();
   }
 
   Set<Marker> _deserializeMarkers(dynamic markers) {
     if (markers == null) {
-      // TODO(iskakaushik): Remove this when collection literals makes it to stable.
-      // https://github.com/flutter/flutter/issues/28312
-      // ignore: prefer_collection_literals
-      return Set<Marker>();
+      return <Marker>{};
     }
     final List<dynamic> markersData = markers;
-    // TODO(iskakaushik): Remove this when collection literals makes it to stable.
-    // https://github.com/flutter/flutter/issues/28312
```

Comment:
Note: ensure this PR **Fixes https://github.com/flutter/flutter/issues/28312** in the description

Reply:
(Done)